packages/grub2/build.yaml

```diff
@@ -5,6 +5,15 @@ requires:
 - name: "cos-config-tmpl"
   category: "utils"
   version: ">=0"
+{{if eq .Values.name "grub2-efi-image"}}
+  {{if .Values.tool_image_distribution}}
+    {{if eq .Values.tool_image_distribution "opensuse"}}
```

Comment:
Is Teal 'opensuse' ?!

Reply:
no, this is the tool image which is used across all flavors to provide system-agnostic packages mainly. So for things like the grub artifacts or efi, we only use one source, which is the artifacts from the tool image directly.

We dont really care about the flavor in here as we generate the same package for all flavors.

Reply:
I kind agree with Klaus this is disturbing... I bet this was done this way because at a time we were not able to `zypper in` into the Teal image the grub packages we needed, the same is happening now with `shim` and `syslinux`.

Reply:
unfortunately this package is only used for raw devices images. Im not even sure if this package should exist or change as its not used anywhere..... we mainly use the grub2-install to generate the grub stuff under elemental-cli so..... I would not change this for now other than adding any extra artifacts that can be added unless we find a use for this.

Reply:
Is this a blocking issue still @kkaempf ?

Reply:
No, and it never was. It's just confusing to have a reference to 'opensuse' here.